README.md

```diff
@@ -1,57 +1,53 @@
 # calculator-angular-tdd
 
-Construyendo una aplicación usando la metodología tdd o test driven development, intentando utilizar las mejores prácticas conocidas por los autores [JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg) 
+Builiding a app using the metology tdd or test driven development, We have used the best practice that thea autors know ([JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg))
 
+We espect have all mecanisme related of continuos integration and continus delivey CI/CD
```

Comment:
espect  -> expect 
We espect have all mecanisme related of -> We expect to have all mechanisms related to ...

Reply:
Done